src/run.ts

```diff
@@ -5,13 +5,14 @@ import * as os from 'os';
 import * as path from 'path';
 import * as util from 'util';
 import * as fs from 'fs';
+import * as semver from 'semver';
```

Comment:
Doesn't this need to be added to package.json?

Reply:
It was already there in node modules along with other packages and non of them are present in package.json. I'm not sure what's the criteria to add modules as dependencies in package.json

Reply:
Should I add it or leave it unchanged?

Reply:
Add it explicitly; we can't take dependencies on other packages to have `semver` all the time! :)

Reply:
I meant it's already present in node modules, but not as a part of another package. 😛